core-build/test-web-library-build/src/test/foo.test.ts

```diff
@@ -1,11 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-/// <reference types="mocha" />
 import { assert } from 'chai';
```

Comment:
>import [](start = 0, length = 6)

Can you show how to import the jsdom polyfill and call it, like Moein was wanting to do?